bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Carousel.java

```diff
@@ -46,6 +46,13 @@
      */
     String PN_AUTOPAUSE_DISABLED = "autopauseDisabled";
 
+    /**
+     * Name of the policy property that defines whether the control elements should be placed in front of the carousel items.
+     *
+     * @since com.adobe.cq.wcm.core.components.models 12.20.0
+     */
+    String PN_CONTROLS_PREPENDED = "controlsPrepended";
```

Comment:
This should be documented in component README

Reply:
addressed in 15801c2b456ec2ea339764304247514deb56967c